test/spec/token.js

```diff
@@ -1099,6 +1835,32 @@ define(function(require) {
       }
     });
 
+    util.itMakesCorrectRequestResponse({
```

Comment:
Also checking (not sure I saw these) - do we have tests for the various error conditions we now throw? I.e. pass in authorizeUrl or userinfoUrl without passing in issuer, id_token vs. access_token.